### PR TITLE
feat: Stabilize Deno.stdin.setRaw()

### PR DESCRIPTION
**Pull request by bartlomieju**

### CODE REVIEW

**Review by lucacasonato — APPROVED** [review]

File: cli/dts/lib.deno.ns.d.ts [hunk 1]
```diff
@@ -1883,9 +1883,7 @@ declare namespace Deno {
    */
   export const File: typeof FsFile;
 
-  /** **UNSTABLE**: new API, yet to be vetted.
-   *
-   *  @category I/O */
+  /** @category I/O */
   export interface SetRawOptions {
     cbreak: boolean;
```
Comment (dsherret) [inline cli/dts/lib.deno.ns.d.ts:1888]: Maybe we should move the docs on the function about this property to instead be on this property?

Comment (bartlomieju) [reply cli/dts/lib.deno.ns.d.ts:1888]: Not sure I understand, can you elaborate?

Comment (dsherret) [reply cli/dts/lib.deno.ns.d.ts:1888]: In other words: maybe we should add some docs directly on this property?

Comment (bartlomieju) [reply cli/dts/lib.deno.ns.d.ts:1894]: Done

**Pull request merged**